scripts/run.js

```diff
@@ -27,13 +27,21 @@ module.exports = async (
     console.log('')
 
     console.log(`Deploying DataFeedOracleBase1...`)
-    const dataFeedOracleBase1 = await DataFeedOracleBase.new()
-    await dataFeedOracleBase1.initialize(owner)
+    const dataFeedOracleBase1 = await TokenPriceDataFeedMock.new()
+    await dataFeedOracleBase1.initialize(
+      "0x0000000000000000000000000000000000000001",
```

Comment:
might consider moving these to constants at the top of the file for readability ... `ONE_ADDRESS`, `TWO_ADDRESS` etc...